packages/headless/src/controllers/core/result-list/headless-core-result-list.test.ts

```diff
@@ -0,0 +1,155 @@
+import {buildMockSearchAppEngine} from '../../../test/mock-engine';
+import {buildMockResult, MockSearchEngine} from '../../../test';
+import {fetchMoreResults} from '../../../features/search/search-actions';
+import {buildCoreResultList, CoreResultList} from './headless-core-result-list';
+import {registerFieldsToInclude} from '../../../features/fields/fields-actions';
+import {configuration, fields, search} from '../../../app/reducers';
+import {SchemaValidationError} from '@coveo/bueno';
+
+describe('CoreResultList', () => {
```

Comment:
It's great that you added this, but it's a bit superfluous 😅 

Or we could remove the tests for `headless-result-list.test.ts` instead, they are just overlapping.

If `headless-result-list` starts getting specific functionalities that are not 100% the same as the core result list, then we'll be able to test for those specifically by adding `headless-result-list.test.ts`

Reply:
Thanks for your input! I wasn't sure of the preferred approach for this project. I'll remove the duplicated tests from `headless-result-list.test.ts`.